testing/create_cluster.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# Create the GKE cluster used for testing Kubeflow deployments.
+# This should be a one-time thing because we reuse the same cluster for all tests.
+gcloud --project=mlkube-testing container clusters create \
+	--zone=us-east1-d  \
+	--machine-type=n1-standard-8 \
+	--cluster-version=1.8.4-gke.1 \
```

Comment:
Pinning the cluster version? we should ideally parse out the `get-server-config` output and pick the latest that way. See https://github.com/kubernetes/test-infra/blob/master/kubetest/extract_k8s.go#L398-L409

Reply:
I removed this file. #72 replaced this script with more complete instructions in the README for setting up the test infrastructure.

The cluster setup is onetime manual. The only reason we pin to a specific version is because I don't know if the default is 1.8 yet. If it is then pinning is unnecessary. Either way, given its a one time manual step there's not much incentive to automating parsing of get-server-config.